src/Microsoft.AspNetCore.SystemWebAdapters.FrameworkServices/Hosting/HttpApplicationHost.cs

```diff
@@ -0,0 +1,53 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
It would actually be really nice to have this as a separate package, perhaps with some of the DI stuff, and maybe even the ContentFile sample extensions in it, that could be used in an ASP.Net 4.x project without SWA at all.
I guess there isn't a huge overhead of pulling in the whole `Microsoft.AspNetCore.SystemWebAdapters.FrameworkServices` package, but if you simply wanted to use the aspire integration part it might be nice to have.